packages/client/src/Loc.ts

```diff
@@ -1326,13 +1335,35 @@ export class OpenLoc extends EditableRequest {
 
     async estimateFeesPublishFile(parameters: { hash: Hash }): Promise<FeesClass> {
         const client = this.locSharedState.client;
-        const file = this.findFile(parameters.hash);
+        const file = this.findFile(parameters.hash, "REVIEW_ACCEPTED");
         return client.estimateFeesPublishFile(file);
     }
 
+    async acknowledgeFile(parameters: AckFileParams): Promise<OpenLoc> {
+        // TODO: Check signer is verified issuer
```

Comment:
To be addressed before release?

Reply:
Addressed by this issue: https://github.com/logion-network/logion-internal/issues/976